react-single-page-app/src/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+
+// <ms_docref_import_modules>
+import { Configuration,  PublicClientApplication } from "@azure/msal-browser";
+import { MsalProvider, useMsal, useIsAuthenticated } from "@azure/msal-react";
+// </ms_docref_import_modules>
+
+// <ms_docref_configure_msal>
+const configuration: Configuration = {
+  auth: {
+      clientId: "034ff080-d093-433d-915c-4f12ae03f100"
+  }
+};
+
+const clientInstance = new PublicClientApplication(configuration);
+// </ms_docref_configure_msal>
+
+// <ms_docref_get_graph_token>
+const getGraphToken =  async (msalInstance, accounts) => {
+  const tokenRequest = {
+    scopes: ["User.Read", "openid", "profile"],
+    account: accounts
+  }
+  
+  try{
+    const {accessToken} = await msalInstance.acquireTokenSilent(tokenRequest);
+    return accessToken;
+  } catch (e) {
+    const {accessToken} = await msalInstance.acquireTokenPopup(tokenRequest);
+    return accessToken;
+  }
+  
+}
+// </ms_docref_get_graph_token>
+
+// <ms_docref_make_graph_call>
+const MICROSOFT_GRAPH_URL = "https://graph.microsoft.com/v1.0"
+
+const fetchUserDetails = async (msalInstance, accounts, setUserDetails) => {
+  const bearer = `Bearer ${await getGraphToken(msalInstance, accounts)}`;
```

Comment:
Inline the code to get the graph token here in this function. (This is learnings/requirements from other examples we have built)